src/Skeletor/Manager/ComposerManager.php

```diff
@@ -28,27 +28,27 @@ public function preparePackageCommand(string $package, string $version, string $
     }
 
     /**
-     * @param string $command
+     * Get the composer file of the project
+     *
+     * @return array
      */
-    public function runCommand(string $command)
+    public function getComposerFile()
     {
-        $this->cli->yellow($command);
-
-        if ($this->options['dryRun']) {
-            return;
-        }
+        return json_decode($this->projectFilesystem->read('composer.json'), true);
+    }
```

Comment:
Maybe check if the json is valid, like how we do this in Cavalry https://github.com/pixelfusion/cavalry-api/blob/master/app/Http/Middleware/JsonApi.php#L28

Reply:
I think it is not really necessary here because this is the fresh composer file of the framework. Chance that it will be invalid is very small I would say ;)